src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
 // write code here
+const fieldBody = document.querySelector('.field').tBodies[0];
+const fieldRows = fieldBody.rows;
+const fieldCells = fieldBody.rows[0].cells;
+
+document.addEventListener('click', (e) => {
+  const button = e.target.closest('.button');
+
+  if (!button) {
+    return;
+  }
+
+  if (button.classList.contains('append-row')) {
+    if (fieldRows.length < 10) {
+      const row = fieldRows[0].cloneNode(true);
+
+      fieldBody.insertAdjacentElement('beforeend', row);
+      document.querySelector('.remove-row').disabled = false;
+
+      if (fieldRows.length === 10) {
+        button.disabled = true;
+      }
+    }
```

Comment:
3 nested `if statements` look not really good. I suggest to figure out how to avoid it